tests/integration/test_model.py

```diff
@@ -77,66 +79,68 @@ def test_training_and_prediction(c, training_df):
         )
     """
     )
-
     check_trained_model(c)
 
 
 @pytest.mark.gpu
 def test_cuml_training_and_prediction(c, gpu_training_df):
-    model_query = """
+    c.sql(
+        """
         CREATE OR REPLACE MODEL my_model WITH (
-            model_class = 'cuml.linear_model.LogisticRegression',
+            model_class = 'LogisticRegression',
             wrap_predict = True,
             wrap_fit = False,
             target_column = 'target'
         ) AS (
             SELECT x, y, x*y > 0 AS target
             FROM timeseries
         )
-        """
-    c.sql(model_query)
+    """
+    )
     check_trained_model(c)
 
 
 @pytest.mark.gpu
 @xfail_if_external_scheduler
 def test_dask_cuml_training_and_prediction(c, gpu_training_df, gpu_client):
-
-    model_query = """
+    c.sql(
+        """
         CREATE OR REPLACE MODEL my_model WITH (
-            model_class = 'cuml.dask.linear_model.LinearRegression',
+            model_class = 'LinearRegression',
             target_column = 'target'
         ) AS (
             SELECT x, y, x*y AS target
             FROM timeseries
         )
-        """
-    c.sql(model_query)
+    """
+    )
     check_trained_model(c)
 
 
 @xfail_if_external_scheduler
 @pytest.mark.gpu
```

Comment:
Don't think this should have too much of an impact, but just noting that `xfail_if_external_scheduler` shouldn't have any impact on a `pytest.mark.gpu` test since they're automatically skipped on our CPU testing